gbmodel/model_sqlalchemy.py

```diff
@@ -182,36 +182,66 @@ def dashboard(self, session_id):
         """
         student = students()
```

Comment:
I was probably too lax in allowing this code in through before as there's no error handling. Could you at least wrap this in a try/except block. Any other cases you can try to catch -- an object perhaps having none for an attribute, etc. would be excellent too.